### ISSUE TITLE
[3.0.2] Altitude and Vario seem to overlap in configurator OSD preview

### ISSUE THREAD

**Issue by 0crap** [iNavFlight/inav-configurator#1351]
Using 3.0.1 firmware I updated to the 3.0.2 Configurator I noticed the Altitude and Vario seem to overlap in the Configurator preview.
While in fact this is not the case.
![Configurator 3 0 2](https://user-images.githubusercontent.com/31951195/133968131-31add8cf-2770-4d98-9944-1153465aa1fe.jpg)

Using 3.0.1 firmware and 3.0.1 Configurator this is shown correctly.
![Configurator 3 0 1](https://user-images.githubusercontent.com/31951195/133968145-6549ac9c-1d68-4f77-86cb-af1314010d84.jpg)

Updating the FC to firmware to 3.0.2 and using the 3.0.2 Configurator also shows these items overlap while this is not the case in the goggle.

**Comment by MrD-RC** [comment]
Yes, latest master 4.0 Configurator and 4.0 firmware, all is fine. Tested on MATEKF405SE target (F405-WING).
